lib/tizen_sdk.dart

```diff
@@ -116,6 +122,36 @@ class TizenSdk {
 
   String get defaultGccVersion => '9.2';
 
+  Future<RunResult> buildNative(
+    String workingDirectory, {
+    @required String configuration,
+    @required String arch,
+    String compiler,
+    String predefine,
+    List<String> extraOptions = const <String>[],
+    Rootstrap rootstrap,
+    Map<String, String> environment,
+  }) async {
+    assert(configuration != null);
+    assert(arch != null);
+
+    return _processUtils.run(<String>[
+      tizenSdk.tizenCli.path,
+      'build-native',
+      '-C',
+      configuration,
+      '-a',
+      arch,
+      '-c',
+      compiler ?? defaultNativeCompiler,
+      if (predefine != null) ...<String>['-d', predefine],
```

Comment:
any cases where multiple predefine values come in?

Reply:
I tested
```
tizen build-native -d "FIRST_DEFINE SECOND_DEFINE THIRD_DEFINE" ...
```
but it didn't work. The predefine option expects only one argument. If you have more than one user defines, they should go in `extraOptions`.

Reply:
```
tizen build-native -d FIRST_DEFINE -d SECOND_DEFINE -d THIRD_DEFINE ...
```
This worked. I'll make a change using `for`.